src/SceneView.tsx

```diff
@@ -120,7 +127,8 @@ export default class SceneView<T extends Route> extends React.Component<
             : focused
             ? StyleSheet.absoluteFill
             : null,
-          style,
+          !focused && defaultStyle,
```

Comment:
Not sure what this does given that nobody is passing the `style` prop and `containerStyle` does the same thing that `style` did but with a different name.

Reply:
i'm honestly not sure how and where the `style` prop is getting its data from, but it does, so I was working around it.

Reply:
The style is prop is modified here https://github.com/callstack/react-native-pager-view/blob/master/src/utils.tsx#L17 by react-native-pager-view.